scss/main.scss

```diff
@@ -492,19 +499,29 @@ h1,h2,h3 {
     font-size: calc(1rem + 27.333%);
     color: $black;
   }
+
+}
+
+#workWithUsText{
+  margin: 0;
 }
 
 .workwithusIconsRow {
   justify-content: space-between;
   text-align: center;
   font-weight: bold;
-  margin: 0 auto 4.39rem;
+  margin: auto 4.39rem;
```

Comment:
This is throwing things off a bit on mobile. Can you please revisit?

<img width="417" alt="Screen Shot 2021-02-18 at 9 01 35 AM" src="https://user-images.githubusercontent.com/9326234/108376227-6366a000-71c8-11eb-92c6-fa66e57494ff.png">

Reply:
I changed the margin to auto which re-centered everything. I believe the icon margins were making them take up too much space on mobile, but it's looking better now.

Reply:
@laurencun Can you take another look? I'm seeing ~90px between the icons on mobile. Figma has it at ~64px.

Reply:
Got it. Should be at 64px on mobile now.